pulsar-zookeeper-utils/pom.xml

```diff
@@ -106,6 +106,13 @@
       <artifactId>simpleclient_caffeine</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>managed-ledger</artifactId>
```

Comment:
Can we avoid introducing manager-ledger as the dependency of `pulsar-zookeeper-utils`?

Reply:
I'd suggest moving EnsemblePlacementPolicyConfig to `pulsar-common`. So it can be used in both `pulsar-zookeeper-utils` and `managed-ledger`.